components/driver/rmt.c

```diff
@@ -538,7 +541,8 @@ static void IRAM_ATTR rmt_driver_isr_default(void* arg)
                 switch(i % 3) {
                     //TX END
                     case 0:
-                        ESP_EARLY_LOGD(RMT_TAG, "RMT INTR : TX END");
+                        if(rmt_on_tx_end_handler == NULL)
+                            ESP_EARLY_LOGD(RMT_TAG, "RMT INTR : TX END");
```

Comment:
I appreciate your consideration for not changing the existing behaviour, but I think it's OK in this case to downgrade this message to ESP_EARLY_LOGV so it only gets logged at the highest "verbose" level. What do you think?

Reply:
You mean downgrading to LOGV and keeping it only when `rmt_on_tx_end_handler ` is used - or keeping as LOGV and always executing (removing the if in line 544)?

Reply:
I meant downgrading to LOGV and always executing - my take is that this is a fairly low-level "verbose" debugging statement. Maybe there's a use case I'm missing, though?